packages/utils/src/lerp.ts

```diff
@@ -1,15 +1,24 @@
 /**
- * Linear interpolation between two numbers.
+ * The `lerp` (linear interpolation) function calculates a value between a
```

Comment:
**thought:** rename the function to `interpolateLinearly`

Reply:
Went originally with `lerp` as it's a standard name for this function.